nixos/modules/system/boot/luksroot.nix

```diff
@@ -860,6 +863,20 @@ in
             });
           };
 
+          activationCommands = mkOption {
+            type = types.lines;
+            default = "";
+            example = ''
+              integritysetup open /dev/sda integ-left
```

Comment:
is this script check somewhere in the module to be valid shell code? Otherwise it could easy to create broken configs with this.

Reply:
Good catch, I've just double-checked my actual usage and it was:
```
      $${cryptsetup}/bin/integritysetup -v open /dev/sda integ-left
      $${cryptsetup}/bin/integritysetup -v open /dev/sdb integ-right
      $${systemd}/bin/udevadm trigger /dev/md126
      $${systemd}/bin/udevadm settle
```

Reply:
Fixed the lack of full Nix store path in the example snippet.

Reply:
> Are you referring to some evaluation-time automated check for the shell snippet validity?

Some scripts are run through bash -n (or whatever the argument to check for valid syntax was) or shellcheck, to make sure they don't immediately fail when being executed.